### DISCUSSION TITLE
Cannot Import more than 1000 new Accounts/Contacts each time

### DISCUSSION THREAD

**Discussion by espe808** [YetiForceCompany/YetiForceCRM#14810] [How to]
Hello,
This is my actual System Configuration.
I cannot import more than 1000 new Accounts/Contacts each time via CSV.
Why?

![fdsfsdfsdfsdfsdf](https://user-images.githubusercontent.com/85027316/121320661-a8484b00-c90d-11eb-80d3-f267d70ffc83.png)

**Comment by vovpff** [comment]
There is limit of records number what can be added at one attempt. If you import more than this limit cron should trigger  and continue importing.

You can change limit in config files.

**Reply by espe808** [reply]
Thanks.
Which file?

**Reply by vovpff** [reply]
I don't remember which one.

Try to check all configuration files. A lot of your questions will dissapear

**Comment by noelhunter** [comment]
Performance.php config file has several settings that may affect an import.  Here are the settings that = 1000

Performance.php:	public static $maxMassDeleteRecords = 1000;
Performance.php:	public static $maxMassTransferOwnershipRecords = 1000;
Performance.php:	public static $CRON_MAX_NUMBERS_SENDING_MAILS = 1000;
Performance.php:	public static $CRON_MAX_ATACHMENTS_DELETE = 1000;

There are other settings in there that may also apply- these are just the ones I saw set to 1000

**Comment by rskrzypczak** [comment]
https://doc.yetiforce.com/code/classes/Config-Modules-Import.html#property_IMMEDIATE_IMPORT_LIMIT
